packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -110,13 +110,26 @@ const getBase64Image = imageProps => {
   }
 
   const loadImage = async () => {
-    const imageResponse = await axios.get(requestUrl, {
-      responseType: `arraybuffer`,
-    })
+    console.log(`Fredwin: Fetching contentful base64 image ${requestUrl}`)
+
+    let imageResponse, body
+    try {
+      imageResponse = await axios.get(requestUrl, {
+        responseType: `arraybuffer`,
+      })
 
-    const base64 = Buffer.from(imageResponse.data, `binary`).toString(`base64`)
+      const base64 = Buffer.from(imageResponse.data, `binary`).toString(
+        `base64`
+      )
 
-    const body = `data:image/jpeg;base64,${base64}`
+      body = `data:image/jpeg;base64,${base64}`
+    } catch (e) {
+      console.log(
```

Comment:
Would it make sense to log `e` here too, or is this definitely always a timeout?

Reply:
Good idea, will do